File: moban_java/src/main/java/org/jdk/project/dto/user/CreateUserRequest.java
package org.jdk.project.dto.user;

import jakarta.validation.constraints.NotBlank;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class CreateUserRequest {
  @NotBlank private String username;
  @NotBlank private String password;
  private String nickname;
  private String email;
  private String phone;
  private String sex;
  private String avatarUrl;
  private Boolean enable;
  private String remark;
}

File: backend/src/main/kotlin/org/jdk/project/dto/user/CreateUserRequest.kt
package org.jdk.project.dto.user

import jakarta.validation.constraints.NotBlank

data class CreateUserRequest(
    @field:NotBlank
    var username: String? = null,
    @field:NotBlank
    var password: String? = null,
    var nickname: String? = null,
    var email: String? = null,
    var phone: String? = null,
    var sex: String? = null,
    var avatarUrl: String? = null,
    var enable: Boolean? = null,
    var remark: String? = null,
)
